File: 217-contains-duplicate/contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        map<int,int>m;
        for(auto num : nums)
        {
            cout<<num<<endl;
            if(m.find(num) != m.end())
                return true;
            else
            {
                m[num]+=1;
            }
        }
        return false;
    }
};

File: 217-contains-duplicate/contains-duplicate.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
        HashSet<int> set = new HashSet<int>();
        int n = nums.Length;
        for(int i = 0; i<n; i++)
        {
            int currentNumber = nums[i];
            if(set.Contains(currentNumber))
            {
                return true;
            }
            set.Add(currentNumber);
        }
        return false;
    }
}